site/docs/v1/tech/identity-providers/google.adoc

```diff
@@ -110,23 +110,32 @@ The Google Client Id found in your Google credentials settings in the `Client ID
 [field]#Client secret# [required]#Required#::
 The Google Client Secret found in your Google credentials settings in the `Client secret` field.
 
-[field]#Button text# [required]#Required#::
-The text to be displayed in the button on the login form. This value is defaulted to `Login with Google` but it may be modified to your preference.
-
 [field]#Login Method# [optional]#Optional#::
 User interaction behavior after clicking on the IdP button (`Login with Google`, for instance).
 +
-* `Use redirect for login` - if selected, the user is redirected to a Google login page. Once authenticated, the user is redirected back to FusionAuth.
+ * `Use redirect for login` - if selected, the user is redirected to a Google login page. Once authenticated, the user is redirected back to FusionAuth.
 If selected, the redirect URL *must* be set to an absolute URL in the Google console for your application. If your hostname is login.piedpiper.com, the redirect URL would be `https://login.piedpiper.com/oauth2/callback/`.
-+
-* `Use popup for login` - if selected, a popup is displayed to the user to login with google. Once authenticated, the window is closed.
+ * `Use popup for login` - if selected, a popup is displayed to the user to login with google. Once authenticated, the window is closed.
 If selected, the `Authorized JavaScript origins` URL *must* be allowed for your host name in the Google console for your application. For example, `\https://login.piedpiper.com`.
 +
 Please note if an `idp_hint` is appended to the OAuth Authorize endpoint, then the interaction behavior will be `redirect`, even if popup interaction is explicitly configured.
 +
+[since]#Since 1.44.0#
+The `Use popup for login` method will render the button in an `iframe` because of changes to the Google JavaScript API.
++
+**If you are using a version of FusionAuth < 1.44.0**, `Use popup for login` won't work after Mar 31 2023. Please see https://github.com/FusionAuth/fusionauth-issues/issues/1939[Issue #1939] for more. This https://fusionauth.io/community/forum/topic/2329/upcoming-google-identity-provider-changes[forum post] has more details on an available workaround and upgrade process.
```

Comment:
```suggestion
**If you are using a version of FusionAuth older than 1.44.0**, `Use popup for login` won't work after Mar 31 2023. Please see https://github.com/FusionAuth/fusionauth-issues/issues/1939[Issue #1939] for more. This https://fusionauth.io/community/forum/topic/2329/upcoming-google-identity-provider-changes[forum post] has more details on an available workaround and upgrade process.
```

Reply:
I thought you were going to put this warning in the API doc as well next to the `loginMethod` field?